go.mod

```diff
@@ -18,11 +21,13 @@ require (
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	github.com/telepresenceio/telepresence/rpc/v2 v2.3.7
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
+	github.com/telepresenceio/telepresence/rpc/v2 v2.3.6
```

Comment:
*Downgrading* `rpc/v2` is a little weird... I suspect it's the artifact of a bad rebase?